front/buildingCache.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+var LimitedEntryMap = require('./LimitedEntryMap.js');
+
+// {id: string, buffer: ArrayBuffer, metadata: Metadata}
+module.exports = new LimitedEntryMap(10000);
```

Comment:
This should be inlined in serverCommunication.
We can change temporarily serverCommunication's API if we need to expose things like the cache size.